### DISCUSSION TITLE
help understanding tkdial.ScrollKnob.set()

### DISCUSSION THREAD

**Discussion by stlamoureux1** [Akascape/TkDial#7] [Q&A]
Hello,

Much appreciation for the work on this package which has really helped streamline a personal project. However, I am stuck on getting the setter method for ScrollKnob to behave as expected. The Dial class renders directly the value entered, but calling scroll_knob.set(value=x) appears to show 2*scroll_knob.end - x. For example, the below snippet displays "190%". I'm new to Tkinter and GUI dev, and not sure what the intended behavior is, so any help understanding the functionality here is much appreciated. Thanks. 

import tkinter as tk
import tkdial

root = tk.Tk()

scroll_knob = tkdial.ScrollKnob(root, end=100)
scroll_knob.pack()

scroll_knob.set(value=10)

root.mainloop()

**Comment by Akascape** [comment]
Ok, its a bug i didn't noticed. I will fix it later.

**Reply by stlamoureux1** [reply]
Thanks.
